zest/releaser/prerelease.py

```diff
@@ -127,6 +132,24 @@ def _grab_history(self):
         # TODO: add line number where an extra changelog entry can be
         # inserted.
 
+        # Look for 'Nothing changed yet' under the latest header.  Not
+        # nice if this text ends up in the changelog.  Did nothing happen?
+        start = headings[0]['line']
```

Comment:
Can't `headings` be an empty list? So that `headings[0]` gives an index error?

I wonder: can't we just simply grab the first 10 lines of the changelog, join them together and check for the 'nothing changed' string? It is not something that needs to be very strict, right? It *does* need to be fail-safe and failure-proof. A quick join of the first 10 lines would be enough, I guess.